lib/express-rate-limit.js

```diff
@@ -85,6 +86,18 @@ function RateLimit(options) {
               );
             }
           }
+          if (options.draft_polli_ratelimit_headers) {
```

Comment:
add a ` && !res.headersSent` check here to avoid errors if other middleware does something unexpected.

Reply:
Okay sure!